MULTIPLATFORM.md

```diff
@@ -27,7 +27,7 @@ Run tests for UIKit:
 ```
```

Comment:
Should be merged only https://github.com/JetBrains/compose-multiplatform-core/pull/1946 is merged